src/Components/Note/Note.jsx

```diff
@@ -18,10 +47,14 @@ const Note = (props) => {
   };
 
   const delRestoreIconClickHandler = () => {
-    dispatch({
-      type: props.delAction === "del" ? "deleteNote" : "restoreNote",
-      payload: props,
-    });
+    props.delAction === "del"
+      ? addToTrashOnServer(delNoteConfig)
+      : addNoteOnServer(restoreFromTrashConfig);
+
+    // dispatch({
+    //   type: props.delAction === "del" ? "deleteNote" : "restoreNote",
+    //   payload: props,
+    // });
   };
 
   const editIconClickHandler = () => {
```

Comment:
Instead of giving editIcon you can give editNoteHandler name. Similarly for everything.